helm-chart/binderhub/templates/deployment.yaml

```diff
@@ -120,7 +117,7 @@ spec:
         {{- end }}
         {{- end }}
         {{- with .Values.extraEnv }}
-        {{- . | toYaml | nindent 8 }}
+        {{- include "jupyterhub.extraEnv" . | nindent 8 }}
```

Comment:
This couples a helper function from the `jupyterhub` chart here. We aren't doing that elsewhere in our chart. Can we put this in a different PR so we can discuss that separately?

Reply:
Reverted in a4c368a!

Reply:
The tests of relevance succeeded, but there was an intermittent failure with polling zenodo - restarted jobs.